src/clj/puppetlabs/services/master/file_serving.clj

```diff
@@ -166,26 +166,27 @@
   "Given a map representing the bolt project configuration, return the
   modulepath. If the modulepath is not defined or the configuration map passed
   is nil then return the default modulepath."
-  [project-config]
-  (let [has-modules-key? (contains? project-config :modules)]
-    (if-let [modulepath (parse-modulepath (get project-config :modulepath))]
-      (if has-modules-key?
-        (concat modulepath [".modules"])
-        modulepath)
-      (if has-modules-key?
-        ["modules" ".modules"]
-        default-project-modulepath))))
+  [bolt-builtin-content-dir project-config]
```

Comment:
Docstring here will need an update

Reply:
Yeah, I'm still updating this. This solution fails acceptance tests so am going to push up a new fix soon.